### PR TITLE
Fix masculine in buyer and seller rows

### PR DESCRIPTION
**Pull request by aarexer**
Fix masculine in buyer and seller rows.

Related to #520

### CODE REVIEW

File: src/main/resources/ru/mystamps/i18n/Messages.properties [hunk 1]
```diff
@@ -119,6 +119,7 @@ t_comment = Comment
 t_image = Image
 t_add_more_images_hint = Later you will be able to add additional images
 t_not_chosen = Not chosen
+t_not_chosen_masculine = Not chosen
```
Comment (php-coder) [inline src/main/resources/ru/mystamps/i18n/Messages.properties]: Could you move this translation into the section for series/info page?

Comment (aarexer) [inline src/main/resources/ru/mystamps/i18n/Messages.properties]: I don't understand what i should do
This translation is in Messages.properties, and i use it in series/info.html in two sections - buyers and sellers.
What's wrong?

Comment (php-coder) [inline src/main/resources/ru/mystamps/i18n/Messages.properties]: Translation string in the `Messages.properties` file are grouped by files where these translations used. Look at the comments in this file and just move this translation to another section.

Comment (aarexer) [inline src/main/resources/ru/mystamps/i18n/Messages.properties]: Ok, i understand, sorry:)
Done.

**Comment by php-coder** [comment]
Merged in 0b423b4 commit. Thank you!

**Pull request closed without merging**